terraform/eks/daemon/emf/main.tf

```diff
@@ -63,21 +63,18 @@ resource "aws_eks_node_group" "this" {
 # EKS Node IAM Role
 resource "aws_iam_role" "node_role" {
   name = "cwagent-eks-Worker-Role-${module.common.testing_id}"
-
-  assume_role_policy = <<POLICY
-    {
-      "Version": "2012-10-17",
-      "Statement": [
-        {
-          "Effect": "Allow",
-          "Principal": {
-            "Service": "ec2.amazonaws.com"
-          },
-          "Action": "sts:AssumeRole"
-        }
-      ]
-    }
-    POLICY
+  assume_role_policy = jsonencode({
```

Comment:
Description of the PR isn't clear so can you explain what was the reasons for this change